lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -6095,32 +6272,34 @@ namespace Js
         Assert(inlineCacheIndex < this->GetInlineCacheCount());
         Assert(inlinee);
 
-        if(!this->m_codeGenGetSetRuntimeData)
+        if(this->GetCodeGenGetSetRuntimeData()==nullptr)
         {
             const auto codeGenRuntimeData = RecyclerNewArrayZ(recycler, FunctionCodeGenRuntimeData *, this->GetInlineCacheCount());
-            this->m_codeGenGetSetRuntimeData = codeGenRuntimeData;
+            this->SetCodeGenGetSetRuntimeData(codeGenRuntimeData);
         }
 
-        const auto inlineeData = this->m_codeGenGetSetRuntimeData[inlineCacheIndex];
+        FunctionCodeGenRuntimeData **codeGenGetSetRuntimeData = this->GetCodeGenGetSetRuntimeData();
+        const auto inlineeData = codeGenGetSetRuntimeData[inlineCacheIndex];
         if(inlineeData)
         {
             return inlineeData;
         }
 
-        return this->m_codeGenGetSetRuntimeData[inlineCacheIndex] = RecyclerNew(recycler, FunctionCodeGenRuntimeData, inlinee);
+        return codeGenGetSetRuntimeData[inlineCacheIndex] = RecyclerNew(recycler, FunctionCodeGenRuntimeData, inlinee);
```

Comment:
yes, but it's cached locally:
FunctionCodeGenRuntimeData **codeGenGetSetRuntimeData = this->GetCodeGenGetSetRuntimeData();